tests/TestWebApplication/Startup.cs

```diff
@@ -60,5 +65,13 @@ public virtual void ConfigureRepositories(IServiceCollection services)
             // Inject our repo.
             services.AddSingleton<IFakeVehicleRepository>(stubbedFakeVehicleRepository);
         }
+
+        // Format: <Microservice>_<HTTP Method>_<MethodName>
+        // E.g. : TestMicroservice__Head_GetListingsAsync
+        private string CustomOperationIdSelector(ApiDescription apiDescription)
+        {
+            var methodName = apiDescription.TryGetMethodInfo(out MethodInfo methodInfo) ? methodInfo.Name : Guid.NewGuid().ToString();
+            return $"TestMicroservice_{apiDescription.HttpMethod}_{methodName}_{Guid.NewGuid().ToString()}";

```

Comment:
is there any way we can make this the _default_, that the consumer can (optionally) override or somehow re-use? I feel like otherwise, this line will be copy/pasted into all the microservices...

Reply:
Nope. That's part of the issue here. `OperationId`'s are (currently) _optional_. So I'm not sure why they were getting generated in the first place. 

I don't want them to be generated at all (currently) because I cannot guarantee a unique name for them, _that is human readable_.

[I've asked an SO question about thi](https://stackoverflow.com/questions/60681379/how-is-the-operationid-used-from-an-openapi-schema?noredirect=1#comment107382607_60681379)s -> why are do they exist and how are they used/consumed. If it's some internal code stuff that a human never uses, then sure -> Guid's all the way. If it's a human that calls 'that method' then I don't want a human typing code that calls some method name that is a Guid.

Reply:
Yeah, i know they're optional.. im saying by _default_ can't we just set it, using the code you've got here, which can be _overriden_?

So if the consumer doesn't do anything, they'll automatically get these OperationIds... unless we don't want that?

Reply:
As I said above, I don't want it to be set _by default_. 

> unless we don't want that?

correct. I (currently) don't want that.